lib/mongoid_rails_migrations/active_record_ext/migrations.rb

```diff
@@ -195,6 +195,13 @@ def rollback(migrations_path, steps=1)
         move(:down, migrations_path, steps)
       end
 
+      def rollback_to(migrations_path, target_version)
+        all_versions = get_all_versions
+	rollback_to = all_versions.index(target_version.to_i) + 1
```

Comment:
Indents here looks a bit broken ^^' Everything else seems good !

Reply:
Merged in master, I fixed the indentation and added a validation on `VERSION` presence